File: Proyecto Riojas/Proyecto/Nodo1.vb
Public Class Nodo1
    Class Nodo1
    {
        
        //Variables
        Public int valor;
        Public Nodo1 siguiente;
        Public GrafosConListas lazos;

        //Propiedades
        Public int Valor 
        { 
            Get
            { 
                Return valor; 
            } 
        }
        Public Nodo1 Siguiente 
        {
            Get
     {             return siguiente;        } 
            Set
            { 
                siguiente = value; 
            } 
        }
        Public GrafosConListas Lazos 
        { 
            Get
            { 
                Return lazos; 
            } 
            Set
            { 
                lazos = value; 
            } 
        }
        
        //Constructores
        Public Nodo1(int valor, Nodo1 Siguiente,GrafosConListas lazos)
        {
            this.valor = valor;
            this.Siguiente = siguiente;
            this.lazos = lazos;
        }

    }
End Class

File: Proyecto Riojas/Proyecto Final1/Proyecto Final1/Nodo1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_Final1
{
    class Nodo1
    {
        
        //Variables
        public int valor;
        public Nodo1 siguiente;
        public GrafosConListas lazos;

        //Propiedades
        public int Valor 
        { 
            get 
            { 
                return valor; 
            } 
        }
        public Nodo1 Siguiente 
        {
            get 
     {             return siguiente;        } 
            set 
            { 
                siguiente = value; 
            } 
        }
        public GrafosConListas Lazos 
        { 
            get 
            { 
                return lazos; 
            } 
            set 
            { 
                lazos = value; 
            } 
        }
        
        //Constructores
        public Nodo1(int valor, Nodo1 Siguiente,GrafosConListas lazos)
        {
            this.valor = valor;
            this.Siguiente = siguiente;
            this.lazos = lazos;
        }

    }
}
